src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -95,4 +95,16 @@
      * @return True if can be interrupted, false if not
      */
     boolean canBeInterrupted();
+
+    /**
+     * Resets this task.
+     */
+    void reset();
+
+    /**
+     * Gets the mutex bits. When the | operation result of two mutex bits is 0, they can run
+     * concurrently.
+     * @return The mutex bits which categorizes this task
+     */
+    int getMutexBits();
```

Comment:
This was actually removed before the AI API was merged into master. It was replaced with `canRunConcurrentWith`.